CODEOWNERS

```diff
@@ -458,6 +458,7 @@ Makefile* @cilium/build
 /pkg/flowdebug/ @cilium/proxy
 /pkg/fqdn/ @cilium/fqdn
 /pkg/fswatcher/ @cilium/sig-datapath @cilium/sig-hubble
+/pkg/healthv2/ @cilium/sig-foundations
```

Comment:
`healthv2` reads weird to me. For modules, it'd be `health/v2` (see [here](https://go.dev/blog/v2-go-modules)) but I'm not sure what the way to go is for packages in tree. Is the goal to eventually rename this `health` again?

Reply:
health/v2 is definitely nice, however there is already a `pkg/health` that has nothing to do with this code so that may be confusing.

Perhaps something like healthprovider may be better.

Reply:
Oh! Yeah for some reason I definitely assumed this is version2 of the same pkg >.<